packages/frontend-shared/src/utils/spec-utils.ts

```diff
@@ -4,8 +4,11 @@ import { ComputedRef, Ref, ref, watch } from 'vue'
 import _ from 'lodash'
 
 import type { UseCollapsibleTreeNode } from '../composables/useCollapsibleTree'
+import { decodeBase64Unicode } from './decodeBase64'
 
-const PATH_SEP = /[\/\\]/
+const platform = JSON.parse(decodeBase64Unicode(window.__CYPRESS_CONFIG__.base64Config)).platform
```

Comment:
I don't think we should use `__CYPRESS_CONFIG__` in `frontend-shared`, it's not available in launchpad - it's `app` specific.

How about exporting this function and using it? https://github.com/cypress-io/cypress/blob/10.0-release/packages/app/src/runner/index.ts#L341-L343